### PR TITLE
norouter: init at 0.6.4

### PR DESCRIPTION
**Pull request by blaggacao**
cc @akihirosuda

### CODE REVIEW

File: pkgs/tools/networking/norouter/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    rev = "v${version}";
+    sha256 = "0h5jzxm4fw50781zj76r5ksnxkzsnrygrykpa913v9nd24c09c7m";
+  };
+
+  vendorSha256 = "sha256-DZ2kcNV8AzNogAUTaeus4rz9gCFo0wm306jcz/cAj0M=";
+
+  subPackages = [ "cmd/norouter" ];
+  doInstallCheck = true;
+  installCheckPhase = ''
+    $out/bin/norouter --version | grep ${version} > /dev/null
```
Comment (nrdxp) [inline pkgs/tools/networking/norouter/default.nix:22]: no tests?

Comment (blaggacao) [reply pkgs/tools/networking/norouter/default.nix:22]: Well, there are upstream integration tests: https://github.com/norouter/norouter/tree/master/integration
But I'm not competent enough to implement them in nixos. I would tend to trust them upstream, though. Not sure if that's an acceptable stance.

What about unit-tests? (like: https://github.com/norouter/norouter/tree/master/cmd/norouter) Wouldn't they be already run by the nixpkgs instrumentation?

Comment (nrdxp) [reply pkgs/tools/networking/norouter/default.nix:22]: nevermind, I overlooked the fact that this was just the install checks. That should be sufficient.

**Review by nrdxp — APPROVED** [review]

File: pkgs/tools/networking/norouter/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  '';
+
+  meta = with lib; {
+    description = "NoRouter: IP-over-Stdio. The easiest multi-host & multi-cloud networking ever. No root privilege is required. ";
```
Comment (fabaff) [inline pkgs/tools/networking/norouter/default.nix:26]: E.g., something like that:

```suggestion
    description = "Tool to handle unprivileged networking by using multiple loopback addresses";
```

see https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes

File: pkgs/tools/networking/norouter/default.nix (hunk 1 above)
Comment (fabaff) [inline pkgs/tools/networking/norouter/default.nix:22]:
```suggestion
    runHook preInstallCheck

    $out/bin/norouter --version | grep ${version} > /dev/null

    runHook postInstallCheck
```

Comment (roberth) [reply pkgs/tools/networking/norouter/default.nix:22]: There's https://github.com/NixOS/nixpkgs/issues/175605, but for now it should be as suggested by @fabaff.

**Comment by blaggacao** [comment]
@fabaff Thank you! I folded your suggestions into the original commit.

**Review by nrdxp — APPROVED** [review]

**Pull request merged**